modules/fluid_properties/doc/content/source/fluidproperties/SalineMoltenSaltFluidProperties.md

```diff
@@ -16,7 +16,7 @@ Saline must be installed alongside MOOSE using the `contrib/saline` submodule in
 module to be able to use this fluid property object. To initialize/download the saline submodule:
 
 ```bash
-git submodule update --init contrib/saline
+git submodule update --init --checkout modules/fluid_properties/contrib/saline
```

Comment:
this is a relative path.
the previous path worked in moose/modules/fluid_properties
this one works in moose/

Reply:
so maybe we want to `cd $MOOSE_DIR` first. except we dont export this one

Reply:
Ah good point, I guess that would work within the module. Any recommendations? I could update the language from  `To initialize/download the saline submodule:` to `To initialize/download the saline submodule within the Fluid Properties module:` to clarify to the user where this git init should be executed